scripts/travis_setup.sh

```diff
@@ -37,15 +39,23 @@ git submodule update --init .mason
 mapbox_time "touch_package_json" \
 touch package.json
 
-# Start the mock X server
-if [ -f /etc/init.d/xvfb ] && [ ! -z "${RUN_XVFB}" ]; then
+function mapbox_start_xvfb {
+    if [ ! -f /etc/init.d/xvfb ]; then
+        echo "Error: Could not start Xvfb mock server."
+        exit 1
+    fi
+
     mapbox_time "start_xvfb" \
     sh -e /etc/init.d/xvfb start
     sleep 2 # sometimes, xvfb takes some time to start up
 
     # Make sure we're connecting to xvfb
     export DISPLAY=:99.0
+}
 
+export -f mapbox_start_xvfb
+
+function mapbox_install_mesa_glx {
```

Comment:
This is needed as a separate script for only the QT builds because Mesa is installed in `platform/linux/config.cmake` for Linux builds, right? Is there any way we could consolidate these into a single script that accepts args to install the necessary version of Mesa?